modules/fscloud/variables.tf

```diff
@@ -62,6 +62,6 @@ variable "cbr_rules" {
     }))) }))
     enforcement_mode = string
   }))
-  description = "(Optional, list) List of CBR rules to create."
+  description = "The optional list of context-based restrictions (CBR) rules to create."
```

Comment:
CBR is also not an approved acronym to my knowledge except when referencing the API methods or CLI commands

Reply:
I have allowed CBR mostly because of this: https://cloud.ibm.com/docs/key-protect?topic=key-protect-access-control-with-cbr

> After you set up your IBM Key Protect service instance, you can manage access by using the Key Protect support for [Context-based restrictions (CBR)](https://cloud.ibm.com/context-based-restrictions).